src/db/eventDb.ts

```diff
@@ -0,0 +1,49 @@
+import pg from 'pg'
+import knex from 'knex'
+
+import { EventObject } from '../types'
+import { config } from '../constants'
+
+const db = new pg.Client({
+  connectionString: config.databaseUrl,
+})
+
+export const createConnection = (): Promise<void> => db.connect()
+
+export const addNewEvent = ({ id, ...event }: EventObject): Promise<EventObject> =>
+  db
+    .query('INSERT INTO posted_events VALUES ($1)', [id])
+    .then(() => ({ ...event, id }))
+    .catch(e => {
+      console.error('failed to save events', e)
+      /* If we can't save the event to the database, we also shouldn't send the 
+      message to the telegram as that would lead to it being shown multiple times */
+      process.exit(1)
```

Comment:
We could just `throw e` instead of exiting the app. The handler is wrapped in a try catch so it should work.

Another thing to consider is using `TaskEither` from `fp-ts` to handle errors nicely.